bin/testObservability/helper/shutdown.js

```diff
@@ -0,0 +1,9 @@
+const RequestQueueHandler = require('./requestQueueHandler');
+
+const shutdown = async () => {
```

Comment:
Let's rename file & method names to indicate how this shutdown hook is used. Also a comment explaining it's usage.